config/custom.css

```diff
@@ -17,5 +17,5 @@ body {
 }
 
 .chat-log {
-    font-family: "Lucida Console" !important;
+    font-family: "Verdana" !important;
```

Comment:
The default font is Verdana, so you can just remove this entire block.
Also, give a better commit description such as "Revert chat log font to default"